tests/testthat/helper.R

```diff
@@ -43,56 +43,16 @@ map_numbers_to_new_range <- function(numbers, lower, upper) {
 
 }
 
-#' A helper function to get all files in the ~/R/ folder
+#' A helper function to get all files in a folder with a specific extension as 
+#' defined by a given pattern.
 #' @keywords internal
 
-get_R_files <- function() {
-
-  R_files <- base::list.files(path = base::paste0(base::getwd(), "/../../R"),
-                                pattern = "*.R",
-                                full.names = TRUE)
-
-  return(unlist(R_files))
-
-}
-
-#' A helper function to get all files in the ~/tests/testthat/ folder
-#' @keywords internal
-
-get_test_files <- function() {
-
-  test_files <- base::list.files(path = base::getwd(),
-                                 pattern = "*.R",
-                                 full.names = TRUE)
-
-  return(unlist(test_files))
-
-}
-
-#' A helper function to get all files in the ~/man/ folder
-#' @keywords internal
-
-get_man_files <- function() {
-
-  man_files <- base::list.files(path = base::paste0(base::getwd(), "/../../man"),
-                                pattern = "*.Rd",
-                                full.names = TRUE)
-
-  return(unlist(man_files))
-
-}
-
-#' A helper function to get all files in the ~/vignettes/ folder
-#' @keywords internal
-
-get_vignette_files <- function() {
-
-  vignette_files <- base::list.files(path = base::paste0(base::getwd(), "/../../vignettes"),
-                                     pattern = "*.Rmd",
-                                     full.names = TRUE)
-
-  return(unlist(vignette_files))
-
+get_files <- function(path, pattern) {
```

Comment:
This is basically the same as you did before and not an improvement. I was thinking more in lines of:

get_files(path = base::file.path(wd, "/../../R"), pattern = "*.R") => return the vector instead of the list which is created by list.files.

However still not seeing any added value in a wrapper for one line of code.

Reply:
I don't quite see what change you want here. I want to keep the `get_visR_files()` function since I am thinking of using it for another check on our package. But for the things going into it, I'm pretty neutral to it, just want to get it out of the way :D So do you want me to remove the (redundant) `unlist` or to just replace the `get_files` call directly with the `list.files` one?